js_modules/dagster-ui/packages/ui-core/src/runs/useRunsForTimeline.tsx

```diff
@@ -251,11 +253,13 @@ export const useRunsForTimeline = ({
   const fetchFutureTicks = useCallback(async () => {
     const queryData = await client.query<FutureTicksQuery, FutureTicksQueryVariables>({
       query: FUTURE_TICKS_QUERY,
-      variables: {tickCursor: startSec, ticksUntil: _end / 1000.0},
+      variables: showTicks
+        ? {tickCursor: startSec, ticksUntil: _end / 1000.0}
+        : {tickCursor: startSec, ticksUntil: startSec},
```

Comment:
is the `tickCursor === ticksUntil` to make this effectively no-op? I think we should just skip this call entirely in this case.

Reply:
Yeah agreed - I tried to do that initially, but it turns out that this query also loads workspace / pipeline information that is required down below.  I think no-oping just the ticks part this way works pretty well, but we could also break this out into another query and then skip? I didn't want to impact the main run timeline if I could help it.